articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,171 @@
+Switch(config-if)#switchport access vlan 20    !making interface access port for vlan 10
+Switch(config-if)#exit   !exiting from the interface
+
+Switch(config)#int fa 0/4     !entering the interface port
+Switch(config-if)#switchport mode access      !making the interface fa0/4 an access port
+Switch(config-if)#switchport access vlan 20   !making interface access port for vlan 10
+Switch(config-if)#exit
+
+```
+In the above  configuration ``fa0/3`` and ``fa0/4`` are configured as access ports using command **`switchport mode access`**.Since they belong to vlan 20 command **`switchport access vlan 20`** is used to configure them as access ports within vlan 20.
+
+#### Configurations for trunk port fa0/5
+```
+Switch(config)#int fa 0/5    !entering the interface port
+Switch(config-if)#switchport mode trunk   !making interface a trunk port.
+Switch(config-if)#do write    !saving the running configurations to start-up file
+```
+From the above code interface, fa0/5 is serving as our trunk port. To configure it to serve as a trunk port and not an access port, we use the command **`switchport mode trunk`** in the global interface mode.
+#### Step 3
+Using static  IP addressing  set the IP addresses to  each PC on the network.
+
+![IP addressing](/engineering-education/inter-vlan-routing/pc-ip-configuration.jpg)
+
+#### Step 4
+Configure the router to enable traffic to move from ``VLAN10`` to ``VLAN20``, and for the PCs to communicate, we subdivide the single interface on the router into sub-interfaces which will act as the default gateways for each of the vlans and this will allow the two sub networks to communicate using the single interface.
+```
+Router>enable        !moving from user exec mode to the priviledge exec mode
+Router#config terminal    !moving from priviledge exec mode to the global configuration mode
+Router(config)#int g0/0        !entering on our physical router interface gigabitEthernet 0/0
+Router(config-if)#no shutdown     !activating the interface
+Router(config-if)#int g0/0.10     !first sub interface for vlan 10 on g0/0
+Router(config-subif)#encapsulation dot1q 10       !configuring the sub interface to respond to traffic from vlan 10
+Router(config-subif)#ip add 192.168.1.1 255.255.255.0      !configuring the IP address of the sub interface g0/0.10
+Router(config-subif)#exit       !exiting from the sub interface
+
+Router(config)#int g0/0     !entering our physical router interface
+Router(config-if)#no shutdown    !activating our physical interface
+Router(config-if)#int g0/0.20     !second sub interface for vlan 20 on g0/0
+Router(config-subif)#encapsulation dot1q 20    !configuring the sub interface to respond to traffic from vlan 20
+Router(config-subif)#ip add 192.168.2.1 255.255.255.0      !configuring the IP address of the sub interface g0/0.20
+Router(config-subif)#exit
+Router(config)#do write       !saving our running configurations into the start-up configuration file
```

Comment:
Please check if all these code snippets work properly

Reply:
I did the inter vlan routing myself in the packet tracer and when I tried to test the communication using ping command it worked perfectly so the code snippets are working 
![ping](https://user-images.githubusercontent.com/77575569/111862115-2414a600-8964-11eb-847d-868a9d9b4c3c.png)
![vlan_routing](https://user-images.githubusercontent.com/77575569/111862116-27a82d00-8964-11eb-9ad6-ac6031373f88.png)

Reply:
Great. Thank you for letting me know.

Reply:
Ook...am using inline editor Vs code with markdown extension for writing the article should I switch to a better editor or just stick with Vs code?

Reply:
No, please stick with the VSCode editor (which is recommended). 

I was confused because in your article, you had HTML tags (which doesn't come in a markdown). So, I thought you had converted them to markdown. My bad.